Documentation/Sprint 1/Sprint reflection.tex

```diff
@@ -0,0 +1,258 @@
+\item[] Eva 
+\item[] Gijs 
+\item[] Jaap 
+\end{enumerate}
+
+& 
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] 2 
+\item[] 13 
+\item[] 8 
+\end{enumerate} 
+ & \begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] 1 
+\item[] 12 
+\item[] 6 
+\end{enumerate}
+
+ & \begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] Yes
+\item[] Yes
+\item[] Yes
+\end{enumerate}
+
+ & \begin{enumerate}[leftmargin=0.5cm,itemindent=0.1cm]
+\item Jan-Willem took over this issue.
+\item 
+\item Gijs took over this issue.
+\end{enumerate}
+
+\\
+
+As a Developer, I want to be able to run Algorithms on a server-side platform & \begin{enumerate}[leftmargin=0.3cm,itemindent=0.1cm]
+\item 
+\item
+\item 
+\end{enumerate}
+
+& 
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] Eva 
+\item[] Tim
+\item[] Eva 
+\end{enumerate}
+
+& 
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] 3 
+\item[] 3 
+\item[] 10 
+\end{enumerate}
+
+&
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] 3 
+\item[] 4 
+\item[]
+\end{enumerate}
+
+&
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] Yes
+\item[] Yes
+\item[] No
+\end{enumerate}
+
+&
+\begin{enumerate}[leftmargin=0.5cm,itemindent=0.1cm]
+\item Was a team effort with Eva as lead.
+\item Originally chosen for Jersey, but decided to use RESTeasy.
+\item Has been delayed till sprint 2 due to running out of time.
+\end{enumerate}
+\\
+As a Developer, I want to be able to store and use the data provided by Soundcloud. & \begin{enumerate}[leftmargin=0.3cm,itemindent=0.1cm]
+\item 
+\item
+\end{enumerate}
+
+& 
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] Eva 
+\item[] Jan-Willem 
+\end{enumerate}
+
+& 
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] 3 
+\item[] 6 
+\end{enumerate}
+
+& \begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] 10 
+\item[] 2 
+\end{enumerate}
+
+& \begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] Yes
+\item[] Yes
+\end{enumerate}
+
+& \begin{enumerate}[leftmargin=0.5cm,itemindent=0.1cm]
+\item Ended up being a lot bigger than expected as features didn't match one-on-one.
+\item Eva took over this issue.
+\end{enumerate}
+
+ \\
+
+As a Developer, I want to have a well defined interface for any data mining algorithms we will use & \begin{enumerate}[leftmargin=0.3cm,itemindent=0.1cm]
+\item 
+\item
+\end{enumerate}
+
+& 
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] Tim
+\item[] Tim
+\end{enumerate}
+
+& 
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] 8 
+\item[] 6 
+\end{enumerate}
+
+&
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] 8 
+\item[]
+\end{enumerate}
+
+&
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] Yes
+\item[] No
+\end{enumerate}
+
+& \begin{enumerate}[leftmargin=0.5cm,itemindent=0.1cm]
+\item Dependency on the core pom.xml was taking 80\% of the issue time. Had to come with new solutions.
```

Comment:
Percentage doesn't say much. Point is, you already developed the library, and our perfectionism forced us to look into the deep cellars of Maven :wink: